widgets/menu/menu.js

```diff
@@ -47,7 +47,7 @@ function menu({
   var containerNode = utils.getContainerNode(container);
   var usage = 'Usage: menu({container, facetName, [sortBy, limit, cssClasses.{root,list,item}, templates.{header,item,footer}, transformData, hideWhenResults]})';
 
-  if (container === null || facetName === null) {
+  if (!container || !facetName) {
```

Comment:
Yeah I know, we'll have to have a place to write those styleguide/standards somewhere soon. Just a wiki page would do.